ppr-ui/public/config/stub-configuration.json

```diff
@@ -1,6 +1,6 @@
 {
   "API_URL": "http://localhost:2015/api/v1/",
-  "AUTH_URL": "http://localhost:8080/auth/",
+  "AUTH_URL": "/auth/",
```

Comment:
Port can change dynamically when running `npm run serve`, so by default use the relative path.